test/ping/test_json.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+"""
+SPDX-License-Identifier: GPL-2.0-or-later
+Copyright (c) 2024 Georg Pfuetzenreuter <mail+ip@georg-pfuetzenreuter.net>
+"""
+
+from json import loads
+from os import getenv
+from subprocess import Popen, PIPE
+from sys import argv, exit
+
+if getenv('VARIANT') == 'cross-compile':
+  exit(77)
+
+count = 2
+
+if len(argv) > 1:
+  ping = argv[1]
+else:
+  ping = 'builddir/ping/ping'
+
+for arguments in [
+  ['-4', 'localhost'],
+  ['::1'],
+  ['-V'],
+  ['-s5000', '-Mdo', 'opensuse.org'],
```

Comment:
FYI I tried to avoid forcing tests to connect over the internet. But OTOH real `getaddrinfo()` functionality (DNS) cannot be tested without internet.

Reply:
I understand, the problem is that this test needs an address which is _not_ localhost, and I cannot predict which network segment the tests are executed in - hence some host on the internet was my best bet.